helm/mcp-kubernetes/templates/ingress.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
I don't think we want to expose an mcp directly. in any case it would be muster the one that is exposed.
muster accesses the mcps through the service

Reply:
I think we should support both in-cluster muster and local muster consuming the mcp running in the cluster through the ingress. The default is not to render the ingress.